tensorbay/utility/common.py

```diff
@@ -236,3 +237,56 @@ def wrapper(*arg: Any, **kwargs: Any) -> Any:
             return func(*arg, **kwargs)
 
         return wrapper  # type: ignore[return-value]
+
+
+def _get_message_template(file: str) -> str:
+    templates = [
+        " This upload action was interrupted.",
```

Comment:
set these strings to constant?

Reply:
A constant `MESSAGE_TEMPLATES` containing different messages is set following this function.